tests/helpers/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .factories import (
     kspace_from_op,
     image_from_op,
+    batchedSmaps_from_op,
```

Comment:
```suggestion
    batched_smaps_from_op,
```
Lets follow the same naming conventions :)